bluemira/equilibria/solve.py

```diff
@@ -586,15 +588,14 @@ def update_fig(self):
         """
         self.ax.clear()
         self.eq.plot(ax=self.ax)
-        self.ax.figure.canvas.draw()
+        plt.pause(PLT_PAUSE)
```

Comment:
What's the reason for the pause here? Could potentially do with a comment

Reply:
This pause is actually moved up from a line below. It seems this is matplotlib standard practice (before they introduced animations) for updating plots. Introducing animations here is a little more of a refactor than just a simple bugfix though.